### PR TITLE
Upgrade to moon v1.17

### PR DESCRIPTION
**Pull request by milesj**
### Summary of Changes

Upgrading moon. Doing this in a separate PR to figure out the root issues.

Edit: I tested a bunch of tasks except for local mobile dev, since I don't have ios/android stuff setup on my computer.

### Demo or Before/After Pics

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots.

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: .github/workflows/mobile-build.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build-android:
-    if: !github.event.pull_request.draft
```
Comment (milesj) [inline .github/workflows/mobile-build.yml:14]: this was invalid syntax

Comment (milesj) [reply .github/workflows/mobile-build.yml:14]: looks like this should go back to being broken :P

**Comment by Robinnnnn** [comment]
image

looks like pod install failed on ios, will just try to re-run it!

File: .github/workflows/mobile-build.yml (hunk 1 above)
Comment (Robinnnnn) [reply .github/workflows/mobile-build.yml:14]: oh hell yeah, didn't know there was an extension!

**Review by Robinnnnn — APPROVED** [review]
LFG, mobile works beautifully locally! 🐐

**Pull request merged**